tensorboard/webapp/metrics/views/metrics_promo_notice_component.ng.html

```diff
@@ -14,4 +14,14 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-Temporary butter bar content.
+<tb-customization [customizableComponent]="customPromoMessage" class="message">
+  Welcome to new default experience of TensorBoard. Time Series lets you view
+  all visualizations at once, put them side-by-side with pins, and customize
+  colors. Scalars and other plugins are still available.
+  <button class="go-to-scalars" (click)="onGoToScalars.emit()">
+    Go to Scalars plugin</button
```

Comment:
Can we make this button looks like more clickable?
```
    text-decoration: underline;
    @include tb-theme-foreground-prop(color, link);
```

Reply:
Done.